.github/ISSUE_TEMPLATE/submit_question.yml

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Submit Question
+description: Ask a general question about cuSpatial
+title: "[QST]: "
+labels: ["question"]
```

Comment:
I think questions should have "needs triage" label too.

Reply:
Ah right - in a way, we should add that label to all new issues. I wouldn't consider an issue out of triage until we've decided where we want it (current release, next release, backlog). Thoughts?

Reply:
Added (and saw needs triage already exists on others)